### DISCUSSION TITLE
FIFWiki/docs/Done/BrainOfTheAlmighty/

### DISCUSSION THREAD

**Discussion by giscus** [FIF-Group/FIFWiki#6] [Announcements]
# FIFWiki/docs/Done/BrainOfTheAlmighty/

A Wiki For FIF(Fullstar Institute of Fenglogy at Hsient'ao, Hupei)

https://fif-group.github.io/FIFWiki/docs/Done/BrainOfTheAlmighty/

**Comment by Alex4210987** [comment]
好汉好
